osu.Game/Overlays/AccountCreation/ScreenEntry.cs

```diff
@@ -223,6 +226,12 @@ private void performRegistration()
                         return;
                     }
 
+                    apiState.BindValueChanged(state =>
+                    {
+                        if (state.NewValue == APIState.RequiresSecondFactorAuth)
+                            this.Push(new ScreenEmailVerification());
+                    });
```

Comment:
This is kind of sketchy (bound and never unbound), but maybe it's fine.

Reply:
It's a local bindable though?

Reply:
Yeah, it's fine as long as the screen is never reused, which I think is fine.